CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.20)
+
+set(CMAKE_CXX_STANDARD 23)
+
+PROJECT(OpenCloud)
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -march=native") # Enable SSE/AVX
```

Comment:
Setting `march=native` by default seems pretty unconventional, I'd suggest either specifically enabling the required SSE version or setting `march=x86-64-v3` which would compile to a generic target with AVX2. (or `-v2` if only SSE4.2).

Reply:
Addressed in 0a7fe7869b1ef0b7f90944e172f9b8309e100c48